InMemoryFileSystemUnitTest/DirectoryTests.cs

```diff
@@ -1,43 +1,101 @@
-using Moq;
-using InMemoryFileSys;
+using NUnit.Framework;
+using System;
 
-namespace InMemoryFileSystemUnitTest
+namespace InMemoryFileSys.Tests
 {
+    [TestFixture]
     public class DirectoryTests
     {
-        private IDirectory _rootDirectory;
-        private Mock<IClock> _clockMock;
+        private Directory _rootDirectory;
 
         [SetUp]
         public void Setup()
         {
-            var mockClock = new Mock<IClock>();
-            mockClock.Setup(clock => clock.UtcNow).Returns(new DateTime(2024, 4, 1));
+            _rootDirectory = Directory.Root;
+        }
+
+        [Test]
+        public void AddFile_ValidRelativePath_FileAddedSuccessfully()
+        {
+            var relativePath = "test.txt";
+
+            var file = _rootDirectory.CreateFile(relativePath);
+
+            var path = file.Path;
+            Assert.IsNotNull(path);
+            Assert.IsTrue(path.EndsWith(relativePath));
```

Comment:
it is better to see the clear output in unit tests, to make less assumptions by a reader. If it is reasonable and easy to do.
here it could be `Assert.That(path, It.Is.Equal("/test.txt"));`